test/e2e/lib/components/support-search-component.js

```diff
@@ -42,6 +47,32 @@ class SupportSearchComponent extends AsyncBaseContainer {
 		);
 	}
 
+	async getDefaultResults() {
+		return await this.driver.findElements( defaultResultsSelectors );
+	}
+
+	async getDefaultResultsCount() {
+		const results = await this.getDefaultResults();
+		return results.length;
+	}
+
+	async waitForDefaultResultsNotToBePresent() {
+		return driverHelper.waitTillNotPresent( this.driver, defaultResultsSelectors );
+	}
+
+	async getErrorResults() {
+		return await this.driver.findElements( errorResultsSelectors );
+	}
+
+	async getErrorResultsCount() {
+		const results = await this.getErrorResults();
+		return results.length;
```

Comment:
Worth checking whether `.findElements` _always_ returns an object that has a length. If not you need a fallback.

Reply:
From the [Selenium docs](https://www.selenium.dev/documentation/en/getting_started_with_webdriver/locating_elements/), it looks like it will return an empty array (they use "list", which I took to mean "array"):

> findElements(By). This method returns a collection of web elements. If only one element is found, it will still return a collection (of one element). If no element matches the locator, an empty list will be returned